hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java

```diff
@@ -55,6 +54,11 @@ public JdbcBackendResolver(FQNResolver fqnResolver) {
         dbType = JDBC_EVENT_PREFIX;
       }
 
+      Optional<String> optionalDbType = DbTelemetryAttributeUtils.getDbTypeForOtelFormat(event);
+      if (optionalDbType.isPresent() && !StringUtils.isEmpty(optionalDbType.get())) {
+        dbType = optionalDbType.get();
```

Comment:
This could be `mongo` too. This should be from a set of SQL values, right?

Reply:
I think you are taking care that `isSqlBackend` is true so, it will be from SQL set, right?

Reply:
yes right